lib/hammer_cli/output/definition.rb

```diff
@@ -73,7 +73,7 @@ def fields_data(fields, sets)
 
         data << sets.each_with_object({}) do |set, res|
           mark = field.sets.include?(set) ? 'x' : ''
-          value = set == sets.first ? field.full_label : mark
+          value = set == sets.first ? field.full_label.capitalize : mark
```

Comment:
This is due to misunderstandings and misusage of the `--fields` option when there e.g. `ID` in help output, but should be used `Id` or `id` instead.